app/views/green_lanes/check_your_answers/show.html.erb

```diff
@@ -11,14 +11,16 @@
                edit_path: 'EDIT GOOD DETAILS PAGE'
     %>
 
-    <h2 class="govuk-heading-m">Category exemptions</h2>
+    <% unless @category_one_assessments.empty? || @answers.nil? %>
```

Comment:
Again if the user has not answered anything then we dont show this section

Reply:
yeah, I think I made a mistake in my  logic